xml/containers-get.xml

```diff
@@ -471,9 +471,9 @@
    updated with the latest security fixes. While the &suse; Registry can be
    used free of charge, if offers additional functionality through a customer
    subscription on the container host. A subscription is required for the
-   container images to receive updates. Currently, the &suse; Registry does not
-   have a UI or a list of images. However, all available images are listed in
-   the examples in the <xref linkend="Building-Pre-build-Images"/>.
+   container images to receive updates. The &suse; Registry has a UI with a
+   subset of images. However, all available images are listed in the examples
+   in the <xref linkend="Building-Pre-build-Images"/>.
```

Comment:
```suggestion
   container images to receive updates. The &suse; Registry's Web user interface
   lists a subset of the available images. Instructions on how to list all available images
   can be found under "Obtaining Base SLES Images" in the <xref linkend="Building-Pre-build-Images"/>.
```

Reply:
Actually, this PR is for the `container-get.xml` file, which is part of the Container Guide. Since it's my area of responsibility, I'll take care of merging it.

Reply:
Actually, this PR is for the `container-get.xml` file, which is part of the Container Guide. Since it's my area of responsibility, I'll take care of merging it.